packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/entityFeatures.ts

```diff
@@ -191,6 +196,216 @@ function cacheGetNeighborEntityElement(
     return element;
 }
 
+/**
+ * Content edit feature to move the cursor from Delimiter After Entity to the delimiter Before
+ */
+const MoveBeforeDelimiterFeature: BuildInEditFeature<PluginKeyboardEvent> = {
+    keys: [Keys.LEFT],
+    shouldHandleEvent: (event: PluginKeyboardEvent, editor: IEditor) => {
+        return getIsDelimiterAtCursor(event, editor, false /* checkBefore */);
+    },
+    handleEvent(event: PluginKeyboardEvent, editor: IEditor) {
+        const delimiter = cacheDelimiter(event, false /* checkBefore */);
+        if (!delimiter) {
+            return;
+        }
+        const delimiterBefore = delimiter.previousElementSibling?.previousElementSibling;
```

Comment:
Do we also need to check previous sibling is entity?

Reply:
Good point, added a check too.